PIL/Image.py

```diff
@@ -1877,7 +1877,7 @@ def __transformer(self, box, image, method, data,
             xs = float(x1 - x0) / w
             ys = float(y1 - y0) / h
             method = AFFINE
-            data = (xs, 0, x0 + xs/2, 0, ys, y0 + ys/2)
+            data = (xs, 0, x0, 0, ys, y0)
```

Comment:
Looks like `xs/2` was designed to compensate for the `AFFINE` transposition bugs and not required anymore.